base/server/python/pki/server/cli/ca.py

```diff
@@ -731,6 +732,96 @@ def execute(self, argv):
             CACertRequestCLI.print_request(request, details=True)
 
 
+class CACertRequestImportCLI(pki.cli.CLI):
+
+    def __init__(self):
+        super().__init__('import', 'Import certificate request into CA')
+
+    def print_help(self):
+        print('Usage: pki-server ca-cert-request-import [OPTIONS]')
+        print()
+        print('  -i, --instance <instance ID>     Instance ID (default: pki-tomcat)')
+        print('      --csr <path>                 Certificate request path')
+        print('      --format <format>            Certificate request format: PEM (default), DER')
+        print('      --profile <filename>         Bootstrap profile filename')
```

Comment:
The help message says profile filename, but the variables (both here and in the related method in `subsystem`) suggest it is the profile ID - I'm not sure which is right (or if they're the same)?

Reply:
AFAIK they are the same but I would prefer to call "profile ID" to be independent from they way profile are stored.

Reply:
Actually, the existing code is using a `profile_id` parameter name for the bootstrap profile, so regardless there will be some sort of "mismatch". Let me merge this as is for now, but we can update it if we figure out a better way to handle this.